test/functional/api/cas/core.py

```diff
@@ -44,9 +44,9 @@ def __get_core_info(self):
         output_lines = output.stdout.splitlines()
         for line in output_lines:
             split_line = line.split(',')
-            if (split_line[0] == "core"
-                    and (split_line[2] == fs_utils.readlink(self.core_device.path)
-                         or split_line[5] == self.path)):
+            if split_line[0] == "core" and (
+                    split_line[2] == os.path.join("/dev", self.core_device.get_device_id())
```

Comment:
Unless you are creating those earlier, I'm not so sure if you can find device by id in `/dev` main directory. I will look for it in `/dev/disk/by-id`. Or maybe I just don't understand what is happening here. ;)

Reply:
I am not looking for this device in /dev because it is already there - this is only check if device used to create core (exported object) is listed in one of lines of "casadm -L" output, to get other piece of information about exported object from this output